### DISCUSSION TITLE
posts/2022/f2f0dea6/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#11] [Announcements]
# posts/2022/f2f0dea6/

年年岁岁花相似，岁岁年年人不同。

https://blog.gztime.cc/posts/2022/f2f0dea6/

**Comment by zhuoy517** [comment]
不知道说什么，但我不相信只有21是你的黄金时代

**Comment by ioslh** [comment]
hackergame 排名看过来的，年轻且实力强，羡慕啊
